packages/web/src/pages/sign-up-page/pages/CreateLoginDetails.tsx

```diff
@@ -66,6 +78,19 @@ export const CreateLoginDetailsPage = () => {
           />
           <Flex direction='column' h='100%' gap='l'>
             <Flex direction='column' gap='l'>
+              <ReadOnlyField
+                label={messages.handleLabel}
+                value={
+                  <Flex alignItems='center' gap='xs'>
+                    @{handleField.value}
```

Comment:
should we create a "handle" component that can have "isVerified" as a flag? just thinking we will prob need this in like 3 or 4 places

Reply:
yeah I think if we use it in more places we should, but I didn't see any other places where we needed it